lib/index.js

```diff
@@ -23,6 +23,9 @@ class PWMetrics {
     this.opts.disableCpuThrottling = this.opts.disableCpuThrottling || false;
     this.metrics = opts.metrics;
```

Comment:
`this.metrics` is redundant, it's already removed in `sheets` branch. But your point is really good. Can you rebase on `sheets-finalizations` branch and make your changes there. It's kinda new version branch where a lot of changes were done. I hope soon @paulirish accept it and we release all this stuff.